apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -82,16 +91,112 @@ class ConnectionInstructions extends React.Component {
           <li>{applabI18n.makerSetupWebSerialConnectToComputer()}</li>
           <li>{applabI18n.makerSetupWebSerialConnectToBoardButton()}</li>
           <li>
-            <SafeMarkdown
-              markdown={applabI18n.makerSetupWebSerialWindowConnect()}
-            />
+            {applabI18n.makerSetupWebSerialWindowConnect()}
+            <ul className={styles.troubleshootList}>
+              <li>
+                <SafeMarkdown
+                  markdown={applabI18n.makerSetupNoCompatibleDevices({
+                    installInstructionsId: styles.installInstructionsId,
+                  })}
+                />
+              </li>
+            </ul>
           </li>
           <li>{applabI18n.makerSetupWebSerialSuccessfulConnection()}</li>
         </ol>
         <SafeMarkdown
           markdown={applabI18n.makerSetupWebSerialSupportArticle()}
+          openExternalLinksInNewTab={true}
         />
         {connectionState}
+        <CPExpressInstallInstructions />
+      </div>
+    );
+  }
+}
+
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div id={styles.installInstructionsId}>
+        <h2>{applabI18n.makerSetupCPXInstallHeader()}</h2>
+        <ol>
+          <li>
+            <div className={styles.listWithImage}>
+              <div>
+                {applabI18n.makerSetupCPXInstallStep1()}
+                <ul>
+                  <li>{applabI18n.makerSetupWebSerialConnectToComputer()}</li>
+                  <li>
+                    {applabI18n.makerSetupCPXInstallStep1a({
+                      buttonName: RESET_BUTTON_NAME,
+                    })}
+                  </li>
+                  <li>
+                    {applabI18n.makerSetupCPXInstallStep1b({
+                      driveName: BOOT_DRIVE_NAME,
+                    })}
+                    <ul className={styles.troubleshootList}>
+                      <li>
+                        {applabI18n.makerSetupCPXInstallStep1c({
+                          buttonName: RESET_BUTTON_NAME,
+                        })}
+                      </li>
+                      <li>{applabI18n.makerSetupCPXInstallStep1d()}</li>
+                    </ul>
+                  </li>
+                </ul>
+              </div>
+              <figure className={styles.imageContainer}>
```

Comment:
What do you think about adding style to this `figure` element? Maybe a light gray border?

Reply:
Here is a treatment similar to the cards on code.org/maker - I was a little skeptical with just an image alone and without any other content but I think it looks nice! What do you think?
<img width="455" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/24951544/852f4f68-3d7b-4162-8176-817aecc23a96">

Reply:
Looks good! We both decided offline to decrease padding -thanks!